modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json

```diff
@@ -13,7 +13,7 @@
           "task_config": {
             "buckets": "{$.meta.buckets}",
             "fileBucketMaps": "{$.meta.collection.files}",
-            "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}"
+            "orcaExcludedFileExtensions": "{$.meta.collection.meta.orca.orcaExcludedFileExtensions}"
```

Comment:
Alright, one more naming thing that I thought I'd mentioned, then I promise I'm done.

With the left-hand-side being `orcaExcludedFileExtensions` should the right-hand-side be `$.meta.collection.meta.orca.excludedFileExtensions`? As-is, we're setting up a naming scheme where every single variable has as part of its path `orca.orca`. If it's already in the `meta.orca` section, we can drop the second orca from the path.

Reply:
orcaExcludedFileExtensions is following the same naming convention of other variables we have like `orcaDefaultBucketOverride` `orcaDefaultStorageClassOverride`

We should either keep it as it is or make the change in those other variables as well. @bhazuka thoughts?

Reply:
using ExcludedFileExtensions on the right side might make documentation more confusing- you have to update for both orcaExcludedFileExtensions and ExcludedFileExtensions

Reply:
I agree we should be consistent regardless of direction. I am just gonna put this out there for discussion but, since we changed `excludedFileTypes` to `orcaExcludedFileExtensions` and if we follow the naming convention to always be `orca<Something>` for our user configurations in Cumulus, maybe we do not need to nest them under and orca key and we can keep them flat under the meta key, thoughts?

Reply:
well that was what the other teams requested- to put all orca related variables under an `orca` key.

I suggest to keep orca.orcaExcludedFileExtensions for now.

Reply:
I think we should keep the `orca` key on RHS. That's what Abdelhak requested in the card.

Maybe the question I should be asking is why we have `Orca` on the LHS... Seems like its only purpose is to tie into how Cumulus defines variables.

We could remove `Orca` on both sides, and have `"excludedFileExtensions": "{$.meta.collection.meta.orca.excludedFileExtensions}"`
which would keep documentation consistent, group our Orca-specific properties for Cumulus, and not tie our naming schema to Cumulus' definitions.

Reply:
yes  we can go with `"excludedFileExtensions": "{$.meta.collection.meta.orca.excludedFileExtensions}"` 
I will update the naming